docs/articles.bib

```diff
@@ -1226,3 +1226,170 @@ @article{Johnston_2013_Spectrum
   url = {https://link.aps.org/doi/10.1103/PhysRevA.88.062330}
 }
 
+@article{Peres_1996_PPT,
```

Comment:
Could you please arrange these alphabetically? If you look throughout the `bib` file, we have sections labeled `Last name begins with P` and so on.